cypress/features/regression/heading.feature

```diff
@@ -18,9 +18,8 @@ Feature: Heading component
     When I set children to "<children>"
     Then heading children on preview is "<children>"
     Examples:
-      | children                |
-      | mp150ú¿¡üßä             |
-      | !@#$%^*()_+-=~[];:.,?{} |
```

Comment:
Leave this pls

Reply:
I deleted special characters because before the previous refactor of code it wasn't an example. It doesn't work here